plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointWriter.java

```diff
@@ -111,7 +111,8 @@ public void write(CheckpointEntries entries, TrinoOutputFile outputFile)
         RowType metadataEntryType = checkpointSchemaManager.getMetadataEntryType();
         RowType protocolEntryType = checkpointSchemaManager.getProtocolEntryType(protocolEntry.getReaderFeatures().isPresent(), protocolEntry.getWriterFeatures().isPresent());
         RowType txnEntryType = checkpointSchemaManager.getTxnEntryType();
-        RowType addEntryType = checkpointSchemaManager.getAddEntryType(entries.getMetadataEntry(), entries.getProtocolEntry(), writeStatsAsJson, writeStatsAsStruct);
+        // TODO https://github.com/trinodb/trino/issues/19586 Add support for writing 'partitionValues_parsed' field
```

Comment:
Please add a testing resource and corresponding test for a checkpoint written by Trino without `partitionValues_parsed` field.